cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -167,9 +167,10 @@
 
   <property>
     <name>zookeeper.quorum</name>
-    <value>127.0.0.1:2181/${root.namespace}</value>
+    <value>FQDN1:2181,FQDN2:2181/${root.namespace}</value>
```

Comment:
Changed to do both: keep the default config to work on a dev setup, but move the change to the description.